test/evp_test.c

```diff
@@ -166,12 +156,8 @@ static int test_bin(const char *value, unsigned char **buf, size_t *buflen)
     }
 
     /* Otherwise assume as hex literal and convert it to binary buffer */
-    *buf = OPENSSL_hexstr2buf(value, &len);
-    if (!*buf) {
-        fprintf(stderr, "Value=%s\n", value);
-        ERR_print_errors_fp(stderr);
+    if (!TEST_ptr(*buf = OPENSSL_hexstr2buf(value, &len)))
```

Comment:
okay, put it back.  I think there should be a TEST_ wrapper around that function, to output it so it looks like test_info calls, but that's a separate PR.